pmm-app/src/pmm-add-instance/components/AddInstance/AddInstance.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { useTheme } from '@grafana/ui';
+import { Database } from 'shared/components/Elements/Icons/Database';
+import { getStyles } from './AddInstance.styles';
+import { instanceList } from './AddInstance.constants';
+import { Messages } from './AddInstance.messages';
+import { AddInstanceProps, SelectInstanceProps } from './AddInstance.types';
+import { OverflowTooltip } from '../../../shared/components/Elements/OverflowTooltip/OverflowTooptip';
+
+export const SelectInstance: FC<SelectInstanceProps> = ({ type, selectInstanceType, title }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <button
+      className={styles.navigationButton}
+      data-qa={`${type}-instance`}
+      onClick={selectInstanceType(type)}
+      type="button"
+    >
+      <Database />
+      <b className={styles.addInstanceTitle}>
+        <OverflowTooltip>{title}</OverflowTooltip>
+      </b>
+      <span className={styles.addInstance}>{Messages.titles.addInstance}</span>
+    </button>
+  );
+};
+
+const AddInstance: FC<AddInstanceProps> = ({ onSelectInstanceType }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  const selectInstanceType = (type) => () => onSelectInstanceType({ type });
```

Comment:
```suggestion
  const selectInstanceType = (type) => () => onSelectInstanceType(type);
```

Reply:
We're doing it because there could be not only a type, but also some credentials for instance if we add RDS. Agree that it looks not very readable and it will be fixed in task PMM-6793 because it affects all components, not only this page

Reply:
Well, it is readable, but it does not follow the best practices. It's a code style thing.